pkg/kv/kvclient/kvcoord/dist_sender_rangefeed.go

```diff
@@ -55,13 +55,15 @@ type singleRangeInfo struct {
 	token      rangecache.EvictionToken
 }
 
-var useDedicatedRangefeedConnectionClass = settings.RegisterBoolSetting(
-	settings.SystemVisible,
-	"kv.rangefeed.use_dedicated_connection_class.enabled",
-	"uses dedicated connection when running rangefeeds",
-	util.ConstantWithMetamorphicTestBool(
-		"kv.rangefeed.use_dedicated_connection_class.enabled", false),
-)
+// defRangefeedConnClass is the default rpc.ConnectionClass used for rangefeed
+// traffic. Normally it is RangefeedClass, but can be flipped to DefaultClass if
+// the corresponding env variable is true.
+var defRangefeedConnClass = func() rpc.ConnectionClass {
```

Comment:
nit: perhaps spell out default?

Reply:
Leaving as is for consistency with [defRaftConnClass](https://github.com/cockroachdb/cockroach/blob/9b833553c591a15270638b1b39268ad6bc975482/pkg/kv/kvserver/replica_init.go#L50).